includes/definitions/microsemipdsine.yaml

```diff
@@ -0,0 +1,7 @@
+os: microsemipdsine
+text: 'Microsemi PowerDsine Midspan PoE'
+type: network
+icon: microsemi
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.7428.1.1.1.23
```

Comment:
I've checked our stats and I can also see `.1.3.6.1.4.1.7428.1.1.1.22` as `Midspan.  Unit S/N=000000.  App Ver=4.17.05.07, Sep 12 2011, 11:19:52.  BOOT Ver=1.07, Jul  8 2009, 15:34:57` Is it worth changing this detection to just use `.1.3.6.1.4.1.7428.1.1.1.` or just add `.1.3.6.1.4.1.7428.1.1.1.22`?